Storage/Clients.swift

```diff
@@ -37,18 +37,18 @@ public struct RemoteClient: Equatable {
         self.device = json["device"].stringValue
```

Comment:
Yup I think this is supposed to be `.string`. We have so many stringifying methods!